File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Client/src/models/csvIdentificationRequest/csvIdentificationRequest.ts
export class CsvIdentificationRequest {
    public id?: string = "";
    public requesterEntraUserId?: string = "";
    public requesterFirstName?: string = "";
    public requesterLastName?: string = "";
    public requesterDisplayName?: string = "";
    public requesterEmail?: string = "";
    public requesterJobTitle?: string = "";
    public recipientEntraUserId?: string = "";
    public recipientFirstName?: string = "";
    public recipientLastName?: string = "";
    public recipientDisplayName?: string = "";
    public recipientEmail?: string = "";
    public recipientJobTitle?: string = "";
    public reason?: string = "";
    public organisation?: string = "";
    public data?: Uint8Array = new Uint8Array();
    public sha256Hash?: string = "";
    public identifierColumn?: string = "";
    public createdBy?: string = "";
    public createdDate?: Date | undefined;
    public updatedBy?: string = "";
    public updatedDate?: Date | undefined;
}

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Server.Tests.Integration/Models/CsvIdentificationRequests/CsvIdentificationRequest.cs
// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

namespace ISL.ReIdentification.Configurations.Server.Tests.Integration.Models.Lookup
{
    public class CsvIdentificationRequest
    {
        public Guid Id { get; set; }
        public string RequesterEntraUserId { get; set; } = string.Empty;
        public string RequesterFirstName { get; set; } = string.Empty;
        public string RequesterLastName { get; set; } = string.Empty;
        public string RequesterDisplayName { get; set; } = string.Empty;
        public string RequesterEmail { get; set; } = string.Empty;
        public string RequesterJobTitle { get; set; } = string.Empty;
        public string RecipientEntraUserId { get; set; } = string.Empty;
        public string RecipientFirstName { get; set; } = string.Empty;
        public string RecipientLastName { get; set; } = string.Empty;
        public string RecipientDisplayName { get; set; } = string.Empty;
        public string RecipientEmail { get; set; } = string.Empty;
        public string RecipientJobTitle { get; set; } = string.Empty;
        public string Reason { get; set; } = string.Empty;
        public string Organisation { get; set; } = string.Empty;
        public string Filepath { get; set; } = string.Empty;
        public byte[] Data { get; set; } = Array.Empty<byte>();
        public string Sha256Hash { get; set; } = string.Empty;
        public int IdentifierColumnIndex { get; set; }
        public bool HasHeaderRecord { get; set; } = false;
        public string CreatedBy { get; set; } = string.Empty;
        public DateTimeOffset CreatedDate { get; set; }
        public string UpdatedBy { get; set; } = string.Empty;
        public DateTimeOffset UpdatedDate { get; set; }
    }
}
